packages/backend/src/managers/applicationManager.ts

```diff
@@ -550,4 +569,190 @@ export class ApplicationManager {
     // Update task
     taskUtil.setTask(checkoutTask);
   }
+
+  /*
+   *
+   * Pod status
```

Comment:
?

Reply:
This was to separate the old content of applicationManager and the content of environmentManager which has been added, but it will probably not be true in a few changes to the file, I will remove it